src/main/java/appeng/core/features/registries/MovableTileRegistry.java

```diff
@@ -39,8 +40,8 @@
 	private final HashSet<Block> blacklisted = new HashSet<>();
 
 	private final HashMap<Class<? extends TileEntity>, IMovableHandler> Valid = new HashMap<>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "Valid" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS00116)